README.md

```diff
@@ -1,239 +1,126 @@
+<p align="center"><img alt="Ego in action" src="https://www.overture.bio/static/screenshot-21fc2cfc0ac1c3fd9bd7e62196477554.png"></p>
 
-<p align="center">
-    <img alt="arch" title="EGO Architecture" src="/docs/ego-arch.png" width="480">
-</p>
-<p align="center">
-  EGO Architecture
-</p>
+Access to sensitive and valuable information necessitates complex and secure methods to verify users and authorize what data and applications they are allowed to access. [Ego](https://www.overture.bio/products/ego/)  simplifies user management by providing a secure system to authenticate and authorize users of your application. [Ego](https://www.overture.bio/products/ego/) uses well-known single-sign-on identity providers like Google, GitHub, LinkedIn and ORCiD in place of managing usernames and passwords. 
 
-## Features
+Ego is [OAuth 2.0](https://oauth.net/2/) and [OpenID Connect](https://auth0.com/docs/authenticate/protocols/openid-connect-protocol) compliant. It is written in JAVA and uses [Sprint Boot](https://spring.io/projects/spring-boot) and [Spring Security Frameworks](https://spring.io/projects/spring-security). Because it uses [JSON Web Tokens (JWT)](https://jwt.io/) for stateless authorization, it can scale to accommodate many users. See our thorough [documentation](https://www.overture.bio/documentation/ego/) for further details. 
 
-Here are some of the features of EGO:
+## Related Products 
 
-- Single Sign on for microservices
-- User-authentication through Federated Identities such as Google, Facebook, Github (Coming Soon), ORCID (Coming Soon)
-- Uses JWT(Json Web Tokens) for Authorization Tokens
-- Provides ability to create permission lists for users and/or groups on user-defined permission entities
-- Built using well established Frameworks - Spring Boot, Spring Security
+<p align="left"><img alt="Overture overview" src="https://www.overture.bio/static/124ca0fede460933c64fe4e50465b235/a6d66/system-diagram.png"></p>
 
-## Tech Stack
+Overtures' modular architecture allows you to utilize and mix any of our products to fulfill your individual needs. Our core technologies, however, can also work in concert as an end-to-end data management system (DMS) designed to satisfy the needs of modern large-scale genomic research. For more information on our DMS, please see our [DMS documentation](https://www.overture.bio/documentation/dms/).
 
-The application is written in JAVA using Spring Boot and Spring Security Frameworks.
+See the links below for additional information on our other modular solutions:
 
-- [Spring Security](https://projects.spring.io/spring-security/)
-- [JWT (JSON Web Tokens)](https://jwt.io/): This project uses [jjwt library](https://github.com/jwtk/jjwt) for JWT related features.
-- [OpenID Connect](http://openid.net/connect/)
+|Product|Description|
+|---|---|
+|[Ego](https://www.overture.bio/products/ego/)|A stateless authorization and user management service|
```

Comment:
Ego provides statless (JWT) and stateful (API-Key) authorization now, so we should probably drop the `stateless` descriptor. The rest is good.

Reply:
got it!